src/userWallet.ts

```diff
@@ -240,6 +242,79 @@ export abstract class UserWallet {
     return usdValue
   }
 
+  generate2fa = () => {
+    const { secret, uri, qr } = generateSecret({
```

Comment:
some other typescript error to fix

Reply:
I don't see what the error is here? Is it that `qr` is an unused var?